src/app/api/orders/route.js

```diff
@@ -0,0 +1,34 @@
+import { NextResponse } from 'next/server';
+
+import prisma from '../../../prisma/prisma.mjs';
+
+const DEFAULT_TAKE = 10;
+
+export async function GET({nextUrl: {searchParams}}) {
+  var response = {}
+  try {
+    if(searchParams.has('getCount') && searchParams.get('getCount') == 'true') {
+      response['count'] = await prisma.order.count();
+    }
+    const cursor = searchParams.has('cursor') ? searchParams.get('cursor') : 'null';
+    const take = searchParams.has('take') ? parseInt(searchParams.get('take')) : DEFAULT_TAKE;
+    var orders;
+    if (cursor != 'null') {
+      console.log("here");
+      orders = await prisma.order.findMany({
```

Comment:
Great.  This is what I was thinking... but do you need a `where` clause to specify the address? Should be a parameter to this backend.